Source/Urho3D/UI/UISelectable.h

```diff
@@ -22,6 +22,11 @@
 
 #pragma once
 
+/// RakNet libraries include windows headers which may conflict with Color::TRANSPARENT
```

Comment:
This is nothing wrong with UISelectable. Such undefs shall be placed after RakNet includes in `Network/`